### PR TITLE
Remove arrow, achievements and games

### PR DESCRIPTION
**Pull request by friedger**
This PR removes references to arrow, achievements and games. #648 describes in details why.

### CODE REVIEW

File: app/src/main/java/org/gdg/frisbee/android/app/GoogleApiClientFactory.java [hunk 1]
```diff
@@ -26,12 +26,7 @@ private static GoogleApiClient createClient(Context context, boolean withSignIn)
             .addApi(AppIndex.API);
 
         if (withSignIn) {
-            Games.GamesOptions gamesOptions = Games.GamesOptions.builder()
-                .setRequireGooglePlus(true)
-                .setShowConnectingPopup(false).build();
-
             builder.addApi(Plus.API).addScope(Plus.SCOPE_PLUS_PROFILE)
-                .addApi(Games.API, gamesOptions)
                 .addApi(Drive.API).addScope(Drive.SCOPE_APPFOLDER);
```
Comment (tasomaniac) [inline app/src/main/java/org/gdg/frisbee/android/app/GoogleApiClientFactory.java]: We are using this to save game state. I think we can remove it as well. If we want to store something we can use Firebase.

**Comment by tasomaniac** [comment]
LGTM. I have only 1 comment. After fixing the lint errors it would be good to go. 

Do you think we should ask other's opinions on this? Nobody is using arrow game but I really liked achievements. Sad to see them go. :(

My personal opinion is :+1:

**Comment by friedger** [comment]
We discussed that publicly, also in the last hangout. So, gone for now. I'd
say.

Let's create a arrow game app.

**Comment by tasomaniac** [comment]
Cool. I couldn't attend the last hangout. 
Creating a separate game makes really sense. Let's do that.

**Comment by tasomaniac** [comment]
LGTM

**Pull request merged**